modules/core/remove_orphaned.py

```diff
@@ -68,6 +68,38 @@ def rem_orphaned(self):
             logger.print_line("No Orphaned Files found.", self.config.loglevel)
             return
 
+        # === AGE PROTECTION: Don't touch files that are "too new" (likely being created/uploaded) ===
+        min_torrent_age_minutes = self.config.orphaned.get("min_torrent_age_minutes", 30)  # Pull from config, default to 30 min
+        now = time.time()
+        protected_files = set()
+
+        if min_torrent_age_minutes > 0:  # Only apply age protection if configured
```

Comment:
won't this always be true if the default is set to 30 minutes? Probably better to set the default to 0 (disabled).